pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -314,7 +319,31 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 		if err != nil {
 			return reconcile.Result{}, err
 		}
+		// We expect this secret to exist in the same namespace Account CR's are created
+		awsClient, err := awsclient.GetAWSClient(r.client, awsclient.NewAwsClientInput{
+			SecretName: controllerutils.AwsSecretName,
+			NameSpace:  awsv1alpha1.AccountCrNamespace,
+			AwsRegion:  "us-east-1",
+		})
+		if err != nil {
+			reqLogger.Error(err, "Failed to get AWS client")
+			return reconcile.Result{}, err
+		}
 
+		callerIdentityOutput, err := awsClient.GetCallerIdentity(&sts.GetCallerIdentityInput{})
+		if err != nil {
+			reqLogger.Error(err, "Failed to get AWS account parent")
+			return reconcile.Result{}, err
+		}
+		// Move Account to Corresponding OU
+		_, err = awsClient.MoveAccount(&organizations.MoveAccountInput{AccountId: &unclaimedAccount.Spec.AwsAccountID, DestinationParentId: &unclaimedAccount.Spec.LegalEntity.ID, SourceParentId: callerIdentityOutput.Account})
```

Comment:
Thinking about this... if the create is in the account controller but the move is here, what happens if the OU doesn't exist yet?

Reply:
I have no idea, I assumed things would be sequential since I was told to do it this way.

Reply:
I think this function will need to be moved out into the account controller

Reply:
For me it makes more sense to check/create the OU in the accountclaim controller since that is where we will have the logic to move it.

Reply:
Or at least some higher level logic will need to be applied